mws/templates/mws/show.html

```diff
@@ -109,7 +109,7 @@ <h3 class="campl-teaser-title">Manage authorised users and groups</h3>
                 </div>
             </div>
 
-            {% if site.production_service.due_update %}
+            {% if site.production_service.due_update or site.production_service.active and site.test_service.active %}
```

Comment:
Irrespective of operator precedence, it is clearer to the reader if this is bracketed:
```python
if site.production_service.due_update or (site.production_service.active and site.test_service.active)
```

Reply:
So, I forgot that brackets aren't supported in Django templates, so refactor this to be something like:
```
{% with prod_and_test_active=site.production_service.active and site.test_service.active %}
{% if site.production_service.due_update or prod_and_test_active %}
    ...
{% endif %}
{% endwith %}
```